scripts/prepdocs.py

```diff
@@ -6,6 +6,7 @@
 import os
 import re
 import time
+import openai
```

Comment:
nit: external non-stdlib imports belong in the next set of imports. (If you run isort, it'll fix it up).